### PR TITLE
Fix/audio issue kick conference

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

**Comment by xquanluu** [comment]
@vdharashive confirmed this is fixed, audio is good after kicked out from conference

**Comment by vdharashive** [comment]
@xquanluu @davehorton this fix has fixed the RTP issue , this can be merged

**Comment by davehorton** [comment]
@xquanluu is there more work to be done on this one?

**Comment by xquanluu** [comment]
No Dave, please help to merge this.

File: lib/session/call-session.js [hunk 1]
```diff
@@ -2074,6 +2074,9 @@ Duration=${duration} `
     // When this call kicked out from conference, session need to replace endpoint
     // but this.ms might be undefined/null at this case.
     this.ms = this.ms || this.getMS();
+    // Destroy previous ep if it's still running.
+    if (this.ep && this.ep.connected) this.ep.destroy();
```
Comment (davehorton) [inline lib/session/call-session.js:2078]: can we change this to 
```
if (this.ep?.connected) this.ep.destroy()
```

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/session/call-session.js (hunk 1 above)
Comment (xquanluu) [reply lib/session/call-session.js]: fixed

**Pull request merged**